### ISSUE TITLE
[question] Is 'gettext' the same recipe as 'libgettext'?

### ISSUE THREAD

**Issue by jgsogo** [conan-io/conan-center-index#1304]
If they are the same (I think so) we should mark one of them as deprecated:
 * Add a comment to the recipe file at the top
 * Add some message to the user like in `jsonformoderncpp`: https://github.com/conan-io/conan-center-index/blob/master/recipes/jsonformoderncpp/all/conanfile.py

Please, add the same message to the top of the `conanfile.py` in `jsonformoderncpp`

Do you think we should standardize this comment at the top of the recipe file and maybe add it to the knowledge base?

**Comment by gocarlos** [comment]
the deprecated should also be in conan center e.g. https://conan.io/center/jsonformoderncpp/3.7.0/?user=_&channel=_

**Comment by uilianries** [comment]
>  Is 'gettext' the same recipe as 'libgettext'?

Same old case: installer package x library package. Now, using the new x-building feature, we can unify both into an unique recipe.

**Comment by Croydon** [comment]
> Do you think we should standardize this comment at the top of the recipe file and maybe add it to the knowledge base?

Yes, definitely. Over the years this will happen a lot. So we should standardize it to not create confusion.

I think there are two forms of deprecations we need to talk about:
  * deprecations of packages (more urgently)
  * deprecation of package versions - we probably don't want to **--build--** all package versions we ever added forever. Maybe "deprecation" is a too strong name for it. We might still accept pull requests which fix bugs for ancient versions, like we do accept PRs in general for configurations we don't build here
